nav_msgs/OccupancyGrid grid
string frame_id
float32 occupancy_threshold
float32 max_distance
float32 min_pass_through
int32[] hit_counts
int32[] pass_through_counts